theme/font.ts

```diff
@@ -0,0 +1,9 @@
+import { extendTheme } from '@chakra-ui/react'
+
+const theme = extendTheme({
+  fonts: {
+    body: `'Jost', sans-serif`,
+  },
+})
```

Comment:
```suggestion
const theme = extendTheme({
  fonts: {
    body: `'Jost', sans-serif`,
  },
  semanticTokens: {
    colors: {
      primary: "#5A1113",
      secondary: "#6B7D87"
    }
  }
})
```

ai ao invés de ter um arquivo chamado `colors.ts` e usar as cores como `color={colors.default}` é só usar `color="primary"` sem ter que importar nada

[ref](https://chakra-ui.com/docs/styled-system/features/semantic-tokens#theme-example)